docs/DatasourcesContents.md

```diff
@@ -0,0 +1,69 @@
+# Terraform Provider Documentation: Datasources
```

Comment:
Would it make more sense to combine the table of contents for data sources and resources? Most OCI resources have both a TF resource and a data source. I could see this perhaps being a table format with columns for service, resource name, link to resource, link to data source. Just a suggestion... what do you think?

Reply:
It makes sense to me, I'll give it a shot

Reply:
created a new file "Table_of_Contents" that merges the datasources and resources links into one ToC. Hopefully there won't be too much churn going forward -- this is all manual work right now and maintaining it could be a bear. I definitely wouldn't try it with examples!